core/src/main/java/io/grpc/TransportManager.java

```diff
@@ -75,6 +72,12 @@
   public abstract InterimTransport<T> createInterimTransport();
 
   /**
+   * Creates an {@link OobTransportProvider} with a specific authority.
+   */
+  public abstract OobTransportProvider<T> createOobTransportProvider(
+      EquivalentAddressGroup addressGroup, String authority);
```

Comment:
It'll be a bit interesting to see how the LB interacts with the NameResolver to call this, handle errors, etc.